pkg/minikube/cluster/cluster.go

```diff
@@ -212,22 +212,23 @@ func adjustGuestClock(h hostRunner, t time.Time) error {
 	return err
 }
 
-// trySSHPowerOff runs the poweroff command on the guest VM to speed up deletion
-func trySSHPowerOff(h *host.Host) {
+// TrySSHPowerOff runs the poweroff command on the guest VM to speed up deletion
+func TrySSHPowerOff(h *host.Host) error {
```

Comment:
This method should be kept private until some other package needs access to it.

Reply:
agree, lets keep methods private unless others use them.